tvbwidgets/ui/surface_widget.py

```diff
@@ -210,5 +245,91 @@ def on_opacity_change(change):
 
         surface_opacity.observe(on_opacity_change, names='value')
 
-        hbox_cortex_controls = ipywidgets.HBox([surface_type, surface_opacity])
-        self.surface_display_controls.children += hbox_cortex_controls,
+        return surface_opacity, surface_type
+
+    def __prepare_points_controls(self, actor, config):
+        def on_size_change(change):
+            value = change['new']
+            actor.GetProperty().SetPointSize(value)
+            self.output_plot.update_plot()
+
+        size_input = ipywidgets.IntText(value=config.size, description='Size: ', disabled=False,
+                                        layout=ipywidgets.Layout(width='250px'))
+        size_input.observe(on_size_change, names='value')
+
+        return size_input,
+
+
+class SurfaceWidgetMenu(ipywidgets.VBox, TVBWidget):
+    # TODO: Keep this separate class?
```

Comment:
I think it is a good idea to have 2 classes, we should think of the naming schema, as it might be repeated for other widgets also. And expose both in the `api.py`
Maybe `SurfaceWidget` and `SurfaceWidgetBase` ?